va/va_fei.h

```diff
@@ -61,6 +61,102 @@ extern "C" {
 
 /**@}*/
 
+/** \brief Attribute value for VAConfigAttribStats */
+typedef union _VAConfigAttribValStats {
+    struct {
+        /** \brief Max number of past reference frames that are supported. */
+        uint32_t    max_num_past_references   : 4;
+        /** \brief Max number of future reference frames that are supported. */
+        uint32_t    max_num_future_references : 4;
+        /** \brief Number of output surfaces that are supported */
+        uint32_t    num_outputs               : 3;
+        /** \brief Interlaced content is supported */
+        uint32_t    interlaced                : 1;
+        uint32_t    reserved                  : 20;
+    } bits;
+    uint32_t value;
+} VAConfigAttribValStats;
+
+typedef struct _VAPictureFEI
+{
+    VASurfaceID picture_id;
+    /*
+     * see flags below.
+     */
+    uint32_t flags;
+} VAPictureFEI;
```

Comment:
Just curious why is it named with "FEI" substring? why not "VAPictureStat"?

Reply:
Yes, VAPictureStats may be more accurate since it's only used for VAEntrypointStats